modules/n1ql/pages/n1ql-language-reference/transactions.adoc

```diff
@@ -184,194 +186,6 @@ When developing a transaction with an SDK, the transaction may contain a mixture
 
 include::learn:partial$transaction-privileges.adoc[]
 
-== Worked Example
-
-This worked example guides you through a complete Couchbase transaction session using N1QL.
-
-[[preparation]]
```

Comment:
Note that removing this section breaks links in the SET TRANSACTION, SAVEPOINT, ROLLBACK TRANSACTION, COMMIT TRANSACTION and BEGIN TRANSACTION reference pages &mdash; need to fix this as part of [DOC-10501](https://issues.couchbase.com/browse/DOC-10501)